keyboards/kbdfans/kbd75/keymaps/ethan605/keymap.c

```diff
@@ -0,0 +1,62 @@
+#include QMK_KEYBOARD_H
+
+#define _______ KC_TRNS
```

Comment:
These defines aren't necessary, as these are actually defined in the core.  You can safely remove them. 

https://github.com/qmk/qmk_firmware/blob/a0309db983150fd0197eb620a6ba552d90d29f93/quantum/quantum_keycodes.h#L30-L31


```suggestion
```